config/locales/user_mailer/en.yml

```diff
@@ -63,17 +63,18 @@ en:
         email addresses. We recommend that you also change your password.
       subject: New email address added
     email_confirmation_instructions:
+      create_new_account: create a new account.
       first_sentence:
         confirmed: Trying to change your email address?
-        forgot_password: You tried to reset your login.gov password but we don't have
-          an account linked to this  email address. If you'd like to set up a new
-          account linked to this email, confirm your email address below.
+        forgot_password_html: "<p>You tried to reset your login.gov password but we
+          don't have an account linked to this  email address.</p><p>You can request
+          your password using a different email address that is connected to your
+          login.gov account.</p>"
```

Comment:
Looks like it was the same before, but since we're here, can we replace `login.gov` with `%{app_name}` and pass in APP_NAME as a param?